circus-train-core/src/main/java/com/hotels/bdp/circustrain/core/replica/Replica.java

```diff
@@ -89,6 +93,29 @@
     this.housekeepingListener = housekeepingListener;
     replicationMode = tableReplication.getReplicationMode();
     this.tableReplication = tableReplication;
+    this.partitionBatchSize = DEFAULT_PARTITION_BATCH_SIZE;
```

Comment:
NABD but you could inline this on line 77 and then only have the assignment in the constructor below which actually allows you to override the value.